src/main/java/se/sundsvall/invoicesender/integration/db/entity/BatchStatus.java

```diff
@@ -0,0 +1,6 @@
+package se.sundsvall.invoicesender.integration.db.entity;
+
+public enum BatchStatus {
+	READY,
+	MANAGED
```

Comment:
Maybe should be `HANDLED` instead of `MANAGED`? (In part based on the log statement at `InvoiceProcessor#237`)